src/database.clj

```diff
@@ -0,0 +1,63 @@
+(ns database
+  (:require [clojure.java.jdbc :as j]))
+
+(def db {:dbtype "postgresql"
+         :dbname "ttt"
+         :host "localhost"
+         :user "angie"
+         :password "MtNmWUsC"})
```

Comment:
Is there a way to hide this in something like an environment variable?

Reply:
+1  This is a good example of data that should be provided from the outside, because it can change frequently and/or is sensitive.  This comes up all the time with database connection strings, URLs to REST APIs, etc...